tests/spectron/spectronSetup.js

```diff
@@ -40,27 +42,52 @@ class App {
     }
 
     static readConfig(configPath) {
-        return new Promise(function (resolve, reject) {
-            fs.readFile(configPath, function (err, data) {
-                if (err) {
-                    reject(err);
-                }
-                resolve(JSON.parse(data));
+
+        if (!fs.existsSync(configPath)) {
+            return this.copyConfigPath();
+        } else {
```

Comment:
we don't need an `else` here as we `return` inside the `if` condition. please remove it.